QMLComponents/models/term.h

```diff
@@ -81,7 +84,7 @@ class Term
 	QStringList		_components;
 	QString			_asQString;
 	bool			_draggable = true;
-	columnType		_type = columnType::unknown;
+	columnTypeVec	_types = {columnType::unknown};
```

Comment:
_types should be settable in the constructor (and can then have `columnType::unknown' as the default argument.